internal/controller/apiextensions/composition/reconciler.go

```diff
@@ -39,6 +38,7 @@ import (
 
 	v1 "github.com/crossplane/crossplane/apis/apiextensions/v1"
 	"github.com/crossplane/crossplane/apis/apiextensions/v1alpha1"
+	"github.com/crossplane/crossplane/internal/controller/apiextensions/composite"
```

Comment:
I'd prefer to avoid one controller importing code from another controller. Can we move `composite.LatestRevision` somewhere that both controllers could import? Perhaps to the API types package?

Reply:
moved to apis/apiextensions/v1/composition_revision.go